src/components/AddressView.tsx

```diff
@@ -3,18 +3,19 @@ import { useZoraUsername } from "@zoralabs/nft-hooks";
 type AddressViewProps = {
   address: string;
   showChars?: number;
+  selector?: any;
 };
 
 const PREFIX_ADDRESS = "0x";
 
-export const AddressView = ({ address, showChars = 6 }: AddressViewProps) => {
+export const AddressView = ({ address, showChars = 6, selector = {} }: AddressViewProps) => {
   const username = useZoraUsername(address);
 
   const addressFirst = address.slice(0, showChars + PREFIX_ADDRESS.length);
   const addressLast = address.slice(address.length - showChars);
 
   if (username.username?.username) {
-    return <span>{`@${username.username.username}`}</span>;
+    return <span {...selector}>{`@${username.username.username}`}</span>;
```

Comment:
Would prefer doing className={} selector or just adding a wrapper class here. Allowing arbitrary props and classnames is a difficult thing to allow in framework code in my experience. You want to totally remove lower level presentational controls since now people can pass in onClick and such. 
I’m okay with a dynamic classname but it may be better to just add a class to keep things consistent and allows us to easily search for how 3rd parties are styling this since we know the name of the class.